clicache/integration-test2/Cluster.cs

```diff
@@ -70,8 +70,9 @@ private bool StartLocators()
 
             for (var i = 0; i < locatorCount_; i++)
             {
+                var startJmxManager = (i == 0);
```

Comment:
Okay, a few problems here:
i) I'd really like to do this without introducing a boolean parameter, though I understand it may be unavoidable.
ii) We don't need a new startJmxManager here, just pass i == 0 as the last parameter to the locator ctor
iii) This was here before you got to it, but this loop is still broken for multiple locators, because the success variable is broken.  If you start two, and the first one fails but the second one works, you've buried an error.  Please break out of the for loop if success is false.

Reply:
i) You can't start the JmxManager in gfsh independent of a locator.
iii) Good catch!